### DISCUSSION TITLE
How to get the "mod", "class", "attr"... notations?

### DISCUSSION THREAD

**Discussion by NahuelCostaCortez** [mkdocstrings/mkdocstrings#644] [Q&A] [answered]
By default, when using mkdocstrings with the literate navigation file the documentation is displayed as in the gif shown at the beginning of the readme of this repo:

![image](https://github.com/mkdocstrings/mkdocstrings/assets/43174126/3338fcfc-e275-4403-ab63-9cdc15a10f35)

However, at https://mkdocstrings.github.io/reference/ one can see the type of each folder/file and the modules/classes/functions are also described at the top of each page:

![image](https://github.com/mkdocstrings/mkdocstrings/assets/43174126/7a274c52-64e3-435a-8de3-476117961b47)
![image](https://github.com/mkdocstrings/mkdocstrings/assets/43174126/0d99ede9-03bd-48b0-9f32-a6d66aa2221f)

Here the differences can be easily seen.

![image](https://github.com/mkdocstrings/mkdocstrings/assets/43174126/14d724d9-aab1-4d2c-b636-9048fa7813aa)
![image](https://github.com/mkdocstrings/mkdocstrings/assets/43174126/53a8e127-bf3c-47a4-bcb2-192eccf0e439)

So my question is how can that be achieved? Sorry in advance if this is already published somewhere else but I have not found anything.

Thanks,
Nahuel

**Comment by pawamoy** [comment, marked as answer]
You're looking for the `show_symbol_type_heading` and `show_symbol_type_toc` options :slightly_smiling_face: 

- https://mkdocstrings.github.io/python/usage/configuration/headings/#show_symbol_type_heading
- https://mkdocstrings.github.io/python/usage/configuration/headings/#show_symbol_type_toc

**Comment by NahuelCostaCortez** [comment]
Thank you for the prompt reply! I see it is already included in the mkdocs.yml of this repo, however, I cloned it and when executing `mkdocs serve` it is not displayed like that. I am wondering if it has to do with any package installation I´m missing. To make it work I installed the following:

```
pip install mkdocs-coverage
pip install mkdocs-git-committers-plugin
pip install mkdocs-techdocs-redirects
pip install mkdocs-minify-plugin
pip install markdown-callouts
```

Not sure it this is the best way to reproduce the web shown in https://mkdocstrings.github.io/reference/ but I did not see any requirements.txt.

Thank you in advance!

**Reply by pawamoy** [reply]
Seems to lack `pip install mkdocstrings-python`? Though without it you'd get an error. It needs `mkdocstrings-python>=1.8.0`.

You can also do this:

```
pip install mkdocs
pip install $(mkdocs get-deps)
```

...to make sure you get all MkDocs plugins and Markdown extensions.

**Comment by NahuelCostaCortez** [comment]
Yeah, sorry I missed the `pip install mkdocstrings-python` in my previous response. With `pip install $(mkdocs get-deps)` now it works so it must depend on some other extension I guess.

Thanks for your help.
